app/reducers/activity.js

```diff
@@ -91,19 +91,19 @@ export function setActivityModalCurrencyFilters(showCurrencyFilters) {
 // Action Handlers
 // ------------------------------------
 const ACTION_HANDLERS = {
-  [SHOW_ACTIVITY_MODAL]: (state, { modalType, modalProps }) => ({
+  [SHOW_ACTIVITY_MODAL]: (state, { itemType, itemId }) => ({
     ...state,
-    modal: { modalType, modalProps }
+    modal: { itemType, itemId }
   }),
-  [HIDE_ACTIVITY_MODAL]: state => ({ ...state, modal: { modalType: null, modalProps: {} } }),
+  [HIDE_ACTIVITY_MODAL]: state => ({ ...state, item: { itemType: null, itemId: null } }),
```

Comment:
This line needs to be:

`[HIDE_ACTIVITY_MODAL]: state => ({ ...state, modal: { itemType: null, itemId: null } }),`

With the current line I can't close the modal as the new item selector below relies on `state.activity.modal._____`

Reply:
Good catch @JimmyMow  - I've updated that now